stdlib/public/core/ArrayBuffer.swift

```diff
@@ -120,7 +120,7 @@ extension _ArrayBuffer {
   /// Convert to an NSArray.
   ///
   /// O(1) if the element type is bridged verbatim, O(*n*) otherwise.
-  @inlinable
+  @usableFromInline
```

Comment:
Isn't this an ABI break/change? What is motivating this?

Reply:
There's a bunch of discussion of this earlier (and in person with Karoy), but basically we can get away with this because although it doesn't obviously look like it, this is Foundation SPI

Reply:
Interestingly, `@inlinable` → `@usableFromInline` is not an ABI break itself, as long as the old implementation keeps working (which is a rather big "as long as")

Reply:
I do suggest still leaving the comment about it just in case we *do* discover a problem. Whoever comes along and is surprised how this got inlined should know.

Reply:
This needs to be commented along with the relevant versions. Maintainers need to know what the old body of code was and for what Swift versions it could have been inlined into client code. It may even be worth having the old definition around inside an `#if false` block.

But, you never answered the question. What is motivating this? That also should be commented.

Reply:
The motivation is that otherwise we'd have to add a runtime availability check that we statically know will always succeed, since Foundation and the stdlib are built together. I don't think that's actually a useful comment to have in the source.

I can reduce the inlinability change to just the one implementation of _asCocoaArray rather than both though.